### DISCUSSION TITLE
建议新增指定词汇不翻译的功能

### DISCUSSION THREAD

**Discussion by SenkuuChin** [immersive-translate/immersive-translate#394] [Ideas]
### 新功能 | New Feature

1. 指定词汇不反应

### 为什么您想要添加这些功能 | Why you want to add these features

1. 有些是专用词汇，但是往往又被翻译。比如 .NET ，就会被翻译成 。网 ，明显是可以不需要翻译的。而现在没有这个功能不去翻译它。
2. 有些人名等也不需要翻译，但是往往又会被翻译。

### 提议 | Proposal

指定过滤特定词汇

**Comment by versun** [comment]
该功能后面会考虑的哈，可以关注https://github.com/immersive-translate/immersive-translate/discussions/385

**Reply by SenkuuChin** [reply]
好的，非常期待！
